db/migrate/20201208092011_append_alert_statuses_feature.rb

```diff
@@ -0,0 +1,19 @@
+class AppendAlertStatusesFeature < ActiveRecord::Migration[5.2]
+  class MiqProductFeature < ActiveRecord::Base; end
+  class MiqRolesFeature < ActiveRecord::Base; end
+
+  def up
+    return if MiqProductFeature.none?
+
+    alert = MiqProductFeature.find_or_create_by!(:identifier => 'alert')
+    alert_status = MiqProductFeature.find_or_create_by!(:identifier => 'alert_status')
```

Comment:
Just for my clarity, did the alert_status feature exist previously, or is it a new feature?

Reply:
Yup, it's an already existing feature...